Components/RangeProperties/index.js

```diff
@@ -0,0 +1,113 @@
+import Paragraph from "../Paragraph";
+import InputTypeRadio from "../InputTypeRadio";
+import { Container } from "../Container";
```

Comment:
Is there a reason why Container is not a default export? If it was then we could be more consistent with our imports.

Reply:
Changed to a default export.